### ISSUE TITLE
WIRELESS dependencies 

### ISSUE THREAD

**Issue by Dmole** [hardkernel/linux#303]
For RTL8192CU, RTL8723DU, and CONFIG_RTL8812AU are not set (disabling wireless will break the build on the odroidxu4-4.9.y branch).

**Comment by mdrjr** [comment]
Disabling those breaks the build?

**Comment by Dmole** [comment]
No; disabling wireless fails to disable the mentioned drivers which causes unmet dependencies.
